src/stdatamodels/jwst/datamodels/model_base.py

```diff
@@ -55,7 +55,10 @@ def on_save(self, init):
         self.meta.date = Time.now().isot
 
 
-# We may want to deprecate DataModel
-# @deprecate_class(JwstDataModel)
 class DataModel(JwstDataModel):
-    pass
+
+    def __init__(self):
+        super(JwstDataModel, self).__init__()
+
+    def __init_subclass__(cls):
+        warnings.warn("Class DataModel has been renamed JwstDataModel. It will be removed in v 1.5.", DeprecationWarning, 2)
```

Comment:
```suggestion
    def __init_subclass__(cls):
        warnings.warn("Class DataModel has been renamed JwstDataModel. It will be removed in v 1.5.", DeprecationWarning, 2)
```